src/controllers/changeset.controllers.js

```diff
@@ -140,6 +140,5 @@ export const createOrUpdateChangesetFile = async (payload) => {
     );
     // Clear skip label if exists
     await removeLabel(baseOctokit, baseOwner, baseRepo, prNumber, SKIP_LABEL);
-    throw new Error("Error creating or updating changeset file.");
```

Comment:
If we do throw an error here, we need to add additional error handling in the github.webhooks.js file. As it was, when I was trying to set up my local environment and figure out how to configure my settings to get everything to work properly, the thrown error here was making it difficult to identify the source of the error, especially since it was only printing a generic message to the console.

Reply:
From @BigSamu :

> You have a point. But that error, in particular, was intended to throw a problem in the check. The middleware then catches the error I sent and throws a new error you were seeing. What we need is to show a check error. We can investigate how to properly show the error sent by the middleware in something more concise. Can you review this?

Reply:
You have a point. But that error, in particular, was intended to throw a problem in the check. The middleware then catches the error I sent and throws a new error you were seeing. What we need is to show a check error. We can investigate how to properly show the error sent by the middleware in something more concise. Can you review this?

Reply:
I've opened an issue in our repo to address this.